test/e2e/aro.go

```diff
@@ -0,0 +1,69 @@
+//go:build e2e
+// +build e2e
+
+package e2e
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+)
+
+const (
+	wSsh = "99-worker-ssh"
+	mSsh = "99-master-ssh"
+)
+
+// The high level idea of these tests is to definitevily say that a cluster is an ARO cluster.
+var _ = Describe("Get ARO cluster", func() {
```

Comment:
Lets talk about this is a more behavioral approach. Lets use something like `Describe("Verify Attributes of ARO Cluster")

We can use `By` statements such that `By("Getting a list of MachineConfigs")

Followed by `It("should be possible to verify aro master and worker ssh machineconfigs exist")`

Reply:
kind of like this https://github.com/Azure/ARO-RP/blob/master/test/e2e/fips.go#L22-L40

Reply:
Thanks for the review @ross-bryan ! I'll make the updates you've suggested.

Do you have any feedback regarding the tests that are included? Should we put effort into adding any new tests?

Reply:
Looks like its off to a good start. Will cross reference the discussion in the design doc to the e2e code in a future review